### ISSUE TITLE
NVDA announces unnecessary information at the beginning for content present under the 'message text' column and provides incomplete information for content present under the 'size' column: A11y_AzureToolsStorageExplorer_Fileshare_ScreenReader

### ISSUE THREAD

**Issue by kapilvaishna** [microsoft/AzureStorageExplorer#8415]
### Preflight Checklist

- [x] I have installed the [latest version of Storage Explorer](https://github.com/Microsoft/AzureStorageExplorer/releases/latest).
- [x] I have checked existing resources, including the [troubleshooting guide](https://docs.microsoft.com/azure/storage/common/storage-explorer-troubleshooting) and the [release notes](https://github.com/microsoft/AzureStorageExplorer/releases).
- [x] I have searched for [similar issues](https://github.com/Microsoft/AzureStorageExplorer/issues).

### Storage Explorer Version

1.36.2

### Architecture

x64

### Platform

Windows

### OS Version

Windows11

## GitHub Tags:

#A11yMAS; #A11yTCS; #Win32; #DesktopApp; #A11ySev3; #BM_AzureToolsStorageExplorer_Win32_Dec2024; #Azure Tools Storage Explorer; #Screenreader; #WCAG1.3.1; #FTP; #NVDA; #Jaws; #STP; #Element:Table;

## Repro Steps 

1. Turn on NVDA
2. Launch Azure Storage Explorer application.
3. TAB till "Account Management"  in left Navigation and press ENTER key.
4. TAB till any "Subsection" for example 'azure tools Manual testing 1' and select it.
5. TAB till "Open explorer" button and press ENTER key
6. "Explorer" section will open. TAB till selected subscription and expand it
7. TAB till storage account and expand it.
8. TAB till any storage account and expand it.
9. TAB till  "File share" and press ENTER key to expand it.
10. TAB till any file share and press ENTER key.
11. TAB till Table and navigate with arrow key
12. Observe that 

## Actual Experience

NVDA announces unnecessary information at the beginning for content present under the 'message text' column and provides incomplete information for content present under the 'size' column when navigating with only the arrow key

## Expected Experience

NVDA should not announce unnecessary information at the beginning for content present under the 'message text' column and NVDA should also announce unit of size for content present under the 'size' column.

### Actual Experience

test

**Comment by craxal** [comment]
The "unnecessary" content is actually the true content of the message text. It's just encoded. We should be using the unencoded text. The size column is also likely just unformatted information.

**Comment by craxal** [comment]
@kapilvaishna Fix is ready for validation.

**Comment by kapilvaishna** [comment]
@craxal both issues mention in actual are reproducing in build 1.37.0 (104)

NVDA

![Image](https://github.com/user-attachments/assets/72deb0d7-aabc-47b2-806d-45a90a00d56b)

JAWS
![Image](https://github.com/user-attachments/assets/2c5dfd62-a04b-4c7f-ad99-d6ebb6574c0a)

**Comment by craxal** [comment]
@kapilvaishna Sorry, I forgot to send you a link to the new build. The build number you should have should be at least 20250128.2 or later. In the future, can you please report the build number that you test on?

**Comment by kupatkar99** [comment]
Verified the issue in below Environment:
Azure Storage Explorer version:1.38.0-dev(104)
Issue is fixed. Please find below added attachment for reference.
#Closed;

![Image](https://github.com/user-attachments/assets/20b275e8-3b9c-4f38-9347-79d60cafd3cd)
